charts/mailhog/values.yaml

```diff
@@ -94,6 +94,8 @@ outgoingSMTP:
     #   host: "mail2.example.com"
     #   port: "587"   # NOTE: go requires this port number to be a string... otherwise the container won't start
 
+replicaCount: 1
```

Comment:
should be `podReplicas`

Reply:
Wow, that was swift response!
had 3 lines, made error in one of them :facepalm: